### PR TITLE
multiline strings

### PR DESCRIPTION
**Pull request by sashu-shankar**
Properly converts multiline strings into PCL and into the target language

Contributes to https://github.com/pulumi/kube2pulumi/issues/41

### CODE REVIEW

File: pkg/yaml2pcl/yaml2pcl.go [hunk 1]
```diff
@@ -338,9 +353,9 @@ func walkToPCL(v Visitor, node ast.Node, totalPCL io.Writer, suffix string) erro
 				return err
 			}
 		}
-		if n.Value.Type() == ast.LiteralType {
-			return nil
-		}
+		//if n.Value.Type() == ast.LiteralType {
```
Comment (EvanBoyle) [inline pkg/yaml2pcl/yaml2pcl.go:356]: Should this be commented out, deleted, or reverted?

File: pkg/yaml2pcl/yaml2pcl.go [hunk 2]
```diff
@@ -191,6 +191,21 @@ func walkToPCL(v Visitor, node ast.Node, totalPCL io.Writer, suffix string) erro
 
 	tk := node.GetToken()
 	switch n := node.(type) {
+	case *ast.LiteralNode:
```
Comment (EvanBoyle) [inline pkg/yaml2pcl/yaml2pcl.go:194]: Does this handle other sorts of YAML literals (`>` `+`, etc)? https://yaml-multiline.info/

We should definitely add a couple of tests that exercise more of the yaml multiline string spec.

File: pkg/pcl2pulumi/pcl2pulumi_test.go [hunk 3]
```diff
@@ -291,3 +321,13 @@ func TestSpecialCharGo(t *testing.T) {
...
+
+func TestMultiLineGo(t *testing.T) {
+	assertion := assert.New(t)
+
+	pcl, err := ioutil.ReadFile("../../testdata/testDep.pp")
+	assertion.NoError(err)
+
+	_, err = Pcl2Pulumi(string(pcl), "../../testdata/testDep", "go")
+	assertion.NoError(err)
+}
```
Comment (EvanBoyle) [inline pkg/pcl2pulumi/pcl2pulumi_test.go:333]: I think we should validate the generated code in these four tests (check in the .go, .cs, etc files to compare). Perhaps we can circle back with the user who reported the issue and send them the generated code to validate against a real deployment.

Comment (sashu-shankar) [reply pkg/pcl2pulumi/pcl2pulumi_test.go:333]: Sounds good! I'll reach out to him and see if he can validate it

File: pkg/yaml2pcl/yaml2pcl.go (hunk 2 above)
Comment (sashu-shankar) [reply pkg/yaml2pcl/yaml2pcl.go:194]: ast.LiteralNode registers all other literals properly, however does not format them in the intended way. Will take care of this in a separate PR

**Comment by EvanBoyle** [comment]
Before we close this out, let's deploy the lambda and cut a new release of kube2pulumi

**Pull request merged**